web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -74,6 +77,19 @@ export const DEFAULT_GESTURE_PARAMS: GestureParams = {
   }
 }
 
+export function keySupportsModipress(key: KeyElement) {
+  const keySpec = key.key.spec;
+  const modifierKeyIds = ['K_SHIFT', 'K_ALT', 'K_CTRL', 'K_NUMERALS', 'K_SYMBOLS', 'K_CURRENCIES'];
+  for(const modKeyId of modifierKeyIds) {
+    if(keySpec.id == modKeyId) {
+      return true;
+    }
+  }
+
+  // Allows special-formatted keys with a next-layer property to be modipressable.
+  return keySpec.sp > ButtonClasses.normal && keySpec.sp < ButtonClasses.deadkey && !!keySpec.nextlayer;
```

Comment:
Can we use `>=` and `<=` to indicate the actual classes that begin and end the valid range, rather than classes that are _not_ in the valid range?

Or perhaps even `[ButtonClasses.a, ButtonClasses.b].includes(keySpec.sp)` or something like that?

Reply:
We... can, though that's definitely more code (and thus, filesize).